src/tools/ceph_dedup_tool.cc

```diff
@@ -595,10 +593,6 @@ int estimate_dedup_ratio(const std::map < std::string, std::string > &opts,
       return -EINVAL;
     }
   } 
-  i = opts.find("debug");
-  if (i != opts.end()) {
-    debug = true;
-  }
 
```

Comment:
Can you add some code like below instead of removing the debug option?
```
    if (debug) {
      ptr->debug = true;
    }
```

Reply:
added.